cmd/plume/release.go

```diff
@@ -461,7 +461,9 @@ func doAWS(ctx context.Context, client *http.Client, src *storage.Bucket, spec *
 				}
 			}
 			if aws.RegionSupportsPV(region) {
-				publish(imageName)
+				if selectedDistro == "cl" {
```

Comment:
I'd say put this as the first case in the outer `if` statement to prevent even checking that the region supports PV unless it's on CL.